README.md

```diff
@@ -65,6 +65,7 @@ Right now, RATS TLS supports the following instance types:
 | ----------- | --------------------- | -------------------------- | -------------------------- | ----------------------- |
 | 0         | nulltls               | nullattester               | nullverifier               | nullcrypto              |
 | 15        | openssl               | sgx\_la                    | sgx\_la                    | openssl                 |
+| 20        | openssl               | csv                        | csv                        | openssl                 |
 | 35        | openssl               | sev                        | sev                        | openssl                 |
 | 42        | openssl               | sev\_snp                   | sev\_snp                   | openssl                 |
 | 42        | openssl               | tdx\_ecdsa                 | tdx\_ecdsa                 | openssl                 |
```

Comment:
Please add prerequisites about running CSV remote attestation.

Reply:
There are no special prerequisites for CSV remote attestation

Reply:
I have explain how to get relevant patches in README.md